content/00.build/30.zksync-sso/20.features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Features
+description: Learn about the features of ZKsync SSO.
+---
+
+## Passkeys Authentication
+
+Passkeys are a core feature supported by ZKsync SSO that enable secure, passwordless authentication for users interacting with applications on the ZK Chain.
+Biometric authentication methods like Face ID and fingerprint recognition are supported by ZKsync SSO, utilizing the `secp256r1` elliptic curve.
+
+## Sessions
+
+A session is a temporary, limited-permission access to the account through a configurable policy.
+Sessions are explicitly approved by account owners and are managed via session keys.
+A session key owner has the ability to send a *certain restricted set* of transactions from the account without requiring the account owner’s signature.
+
+## Spend Limits
+
+**Spend limits** define the maximum amount of a specific token that can be spent during a session.
+They are an integral part of the session's policy and provide granular control over:
```

Comment:
Spend limits are not an integral part of the session policy.
In fact, we don't use them in most of our demos.

Also, spend limits are not limited to only ERC-20 tokens.